crates/turborepo-repository/src/discovery.rs

```diff
@@ -224,6 +255,28 @@ impl<A: PackageDiscovery + Send + Sync, B: PackageDiscovery + Send + Sync> Packa
             }
         }
     }
+
+    async fn discover_packages_blocking(&self) -> Result<DiscoveryResponse, Error> {
+        tracing::debug!("discovering packages using fallback strategy");
+
+        tracing::debug!("attempting primary strategy");
+        match tokio::time::timeout(self.timeout, self.primary.discover_packages_blocking()).await {
+            Ok(Ok(packages)) => Ok(packages),
+            Ok(Err(err1)) => {
+                tracing::debug!("primary strategy failed, attempting fallback strategy");
+                match self.fallback.discover_packages_blocking().await {
+                    Ok(packages) => Ok(packages),
+                    // if the backup is unavailable, return the original error
+                    Err(Error::Unavailable) => Err(err1),
+                    Err(err2) => Err(err2),
+                }
+            }
+            Err(Elapsed) => {
```

Comment:
I don't think this is doing what's intended. This is creating a variable named `Elapsed` and assigning the error to it. I think this is the correct syntax:
```suggestion
            Err(Elapsed { .. }) => {
```

Reply:
I was worried this was the case, TIL on the syntax, TY